.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -16,20 +23,10 @@ body:
       options:
         - label: 'I have searched and found no existing issue'
           required: true
+        - label: 'I will be providing steps how to reproduce the bug (in most cases this will also mean uploading a demo budget file)'
+          required: true
     validations:
       required: true
-  - type: checkboxes
-    id: bank-sync-issue
-    attributes:
-      label: 'Is this related to GoCardless, Simplefin or another bank-sync provider?'
-      description: 'Most issues with bank-sync providers are due to a lack of a custom bank-mapper (i.e. payee or other fields not coming through). In such cases you can create a custom bank mapper in [actual-server](https://github.com/actualbudget/actual-server/blob/master/src/app-gocardless/README.md) repository. Other likely issue is misconfigured server - in which case please reach out via the [community Discord](https://discord.gg/pRYNYr4W5A) to get support.'
-      options:
-        - label: 'I have checked my server logs and could not see any errors there'
-        - label: 'I will be attaching my server logs to this issue'
-        - label: 'I will be attaching my client-side (browser) logs to this issue'
-        - label: 'I understand that this issue will be automatically closed if insufficient information is provided'
-    validations:
-      required: false
```

Comment:
I think people didn't understand those boxes were meant for only sync issues.  It may be better to have a dedicated ticket type for bank sync instead of including it in the generic "bug" type